ntoskrnl/include/internal/i386/asmmacro.S

```diff
@@ -207,6 +207,9 @@ set_sane_segs:
         mov fs, ax
     endif
 
+    mov eax, fs:[KPCR_EXCEPTION_LIST]
```

Comment:
A comment to recall why we are doing this may be in order...

Reply:
Will add.